### DISCUSSION TITLE
Button 'Search'

### DISCUSSION THREAD

**Discussion by Csavie** [dnGrep/dnGrep#1056] [General]
Hi,

When it has some time that I do not use dnGrep I found difficult to find the button to do the "Search". 

I saw that the default button has a blue border Button.Default.Border, but this is not enough for me. I tried to change this colour but these isn't enough either. Maybe a colour Button.Default.Background or Button.Default.Foreground, or colours for Button.Search, or if the button was bigger or in another position....., I don't know, is only some thinking.

Csavie

**Comment by doug24** [comment]
OK, I'll think about it - but I don't want to get too far off the Windows standards...

**Comment by Csavie** [comment]
Ok, I understand this.
